src/Classes/Password.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Classes;
+
+
+class Password
+{
+
+    public function checkEmail($params)
```

Comment:
Just subjective feedback here, but the name `checkEmail` as a function name implies the return might be boolean, but it's returning the actual email address if it finds a match. A more clear name for this function would be `getEmail`